src/pyodide.js

```diff
@@ -266,6 +268,14 @@ var languagePluginLoader = new Promise((resolve, reject) => {
     return {};
   };
 
+  Module.checkABI = function(ABI_number) {
+    if (ABI_number !== parseInt('{{ABI}}')) {
```

Comment:
Why are we making a string literal, and then converting it so an int?  Couldn't we just do `{{ABI}}` instead of `parseInt('{{ABI}}')`?

Reply:
That's because the ``sed`` command used in the ``Makefile`` that replaces it with the actual number replaces it as a string. I couldn't find how to make it an integer with ``sed``. Hope this suffices. Should I still change it?

Reply:
It shouldn't matter that `sed` is working on strings -- that's at build time, where everything is a string because it's a source code file.  The conversion here is happening at runtime.  It should be possible to make this change.

Reply:
Aah yes, I understand now. But using just ``{{ABI}}`` [without the quotes] is probably not correct according to the javascript syntax (My editor kinda highlights it in red). Although it shouldn't matter as this file will be interpreted as if it is a normal text file by ``sed`` and this file is not used as a ``.js`` file. Can I use ``{ABI}`` instead (which is actually correct according to the javascript syntax)? But then, this file also uses ``{{DEPLOY}}`` above so there will be inconsistency between using ``{ }`` OR ``{{ }}``.

Again - We can of course use just ``{{ABI}}`` [without quotes] but then it's not right if we see this file as a ``.js`` file rather than a text file (as seen by ``sed``).

Lastly, I can also use ``'{{ABI}}'`` [with quotes] but that will also involve introducing the quotes in the ``sed`` command in the ``Makefile`` aka using 
````
sed -i -e "s#'{ABI}'#$(PYODIDE_PACKAGE_ABI)#g" $@ [with quotes]
````
instead of
````
sed -i -e "s#{ABI}#$(PYODIDE_PACKAGE_ABI)#g" $@ [previously without quotes]
````
So what is preferred here?

Reply:
I see your point now.  The file doesn't have to be valid Javascript, since it is always run through `sed` by the build system before it is loaded.  But I suppose it will still confuse some editors.  Why don't we just keep it as we have here for now.

Reply:
Okay, I am then not changing anything and using ``parseInt`` as before if that is what you mean by ``keeping it as we have here``.